### PR TITLE
Design implementation - WIP

### PR DESCRIPTION
**Pull request by aeassaf**
- Added navigation bar
- Added links in home page
- Added Backgrounds to all pages

### CODE REVIEW

File: src/components/Home/index.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
+.background {
+  z-index: 9999;
+  width: 100%;
+  height: 94.5%;
```
Comment (JadTermsani) [inline src/components/Home/index.css]: Use calc(100vh - header height) instead

**Comment by george-haddad** [comment]
You are bundling the binary images in the same folder as your source files (.js ...etc.) it's not a good practice. They should be placed in the `public` folder where it should be. Read this to learn why it is the best thing to do: https://facebook.github.io/create-react-app/docs/using-the-public-folder

**Comment by george-haddad** [comment]
In general the topic of the PR is too broad, tackle smaller pieces of work so that the diffs are smaller and can be evaluated better.

Having multiple running PRs in parallel is completely ok and a sign of advanced programmer.

Hook up to a CI like Circle or Travis to gain automated integration.

Hook up to a CD like netlify to deploy your site and branch previews to the net for visual testing.

**Comment by george-haddad** [comment]
branch name is also not that descriptive and is very vague. You are always implementing a design almost all the time. The branch name should encompass the work you are doing in terms of feature development. The PR description should be more thorough as "What you are doing" and your commits will be "how" you are doing it.

File: src/components/Home/index.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  left: 0;
+  background-image: url(./home.jpg);
+  background-size: cover;
+}
+
+.buttonstyle {
+  padding-left: 100px;
+  padding-right: 100px;
+  padding-top: 20px;
+  padding-bottom: 380px;
```
Comment (JadTermsani) [inline src/components/Home/index.css]: you can use 
`padding: 20px 100px 380px 100px;`

**Review by george-haddad — COMMENTED** [review]
Seems fine, but your circleci is just building (`yarn install`) would be cool for it to lint your project as a proper test before merging.

**Comment by JadTermsani** [comment]
I'll approve for now (just add the linting check on CircleCI as george requested) then Refactor the designs on another branch and PR

**Review by JadTermsani — APPROVED** [review]

**Pull request merged**